scripts/cfevents/capture.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+SPACE=$1
+
+aws s3 rm --recursive s3://$S3_BUCKET/cfevents/$SPACE/
+
+cf run-task cfevents --name cfevents-instance --command "/opt/cfevents/capture-latest-events $SPACE"
+
+sleep 15
```

Comment:
I don't  like this timing assumption, but also I think we'll be able to drop this once cfevents is emitting log messages and those are flowing to NR and the log-shipper's s3 bucket. Let's make some time to talk about this.